playwright/models/DeviceAction.ts

```diff
@@ -0,0 +1,61 @@
+import { Page } from '@playwright/test';
+import {
+  deviceInfo155 as deviceInfo,
+  mockListAppsResult,
+} from "@ledgerhq/live-common/lib/apps/mock";
+
+export class DeviceAction {
+  readonly page: Page;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  async openApp() {
+    await this.page.evaluate(() => {
+      window.mock.events.mockDeviceEvent({ type: "opened" });
+    });
+
+    await this.page.waitForSelector("#deviceAction-loading", { state: "visible" });
+  }
+
+  async genuineCheck() {
+    const result = mockListAppsResult("Bitcoin", "Bitcoin", deviceInfo);
```

Comment:
Why are we hard coding Bitcoin?

Reply:
agreed, pass an arg to `genuineCheck(coin1, coin2)` so you can pass it to `mockListAppsResult(coin1, coin2, deviceInfo)` or something similar

Reply:
I agree, I just copy/pasted what we did on spectron 😄 .